docroot/modules/custom/civictheme_content/modules/civictheme_content_default/content/menu_link_content/02c12690-72aa-4f0d-bd3e-3380ecbebb2e.yml

```diff
@@ -30,7 +30,7 @@ default:
       value: false
   weight:
     -
-      value: 0
+      value: -48
```

Comment:
changed the order of created menu links within a footer sub-menu to make it different from the previous set of links.